### PR TITLE
user profile integration into the circle information page

### PR DESCRIPTION
**Pull request by schalky**
Also fix around schedule of session and DST

### CODE REVIEW

File: lib/models/circle.dart [hunk 1]
```diff
@@ -17,13 +17,17 @@ class Circle {
...
-  Circle.fromJson(Map<String, dynamic> json, {required this.id, UserProfile? createdUser}) {
+  Circle.fromJson(Map<String, dynamic> json,
+      {required this.id, UserProfile? createdUser}) {
     name = json['name'] ?? "";
     description = json['description'];
     createdBy = createdUser;
     createdOn = DateTimeEx.fromMapValue(json['createdOn']) ?? DateTime.now();
     updatedOn = DateTimeEx.fromMapValue(json['updatedOn']);
+    final participantRefs = json['participants'];
+    if (participantRefs != null) {}
```
Comment (blopker) [inline lib/models/circle.dart:30]: TODO?

File: lib/services/firebase_providers/firebase_circles_provider.dart [hunk 2]
```diff
@@ -222,7 +240,7 @@ class FirebaseCirclesProvider extends CirclesProvider {
         debugPrint("unable to create session: " + ex.toString());
       }
     }
-    batch.commit();
+    await batch.commit();
```
Comment (blopker) [inline lib/services/firebase_providers/firebase_circles_provider.dart:243]: Oops :)

**Review by blopker — APPROVED** [review]
Nice!

File: lib/models/circle.dart (hunk 1 above)
Comment (schalky) [reply lib/models/circle.dart:30]: oh, yep, its cruft actually, will remove in next push

**Pull request merged**